meta-lmp-bsp/dynamic-layers/stm-st-stm32mp/recipes-support/stm32-mfgtool-files/stm32-mfgtool-files/provision.sh.in

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
```

Comment:
This script has been re-worked.  I need to sync up with @igoropaniuk to show him the changes.